.env

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_TITLE_URL='famf.app'
```

Comment:
`NEXT_PUBLIC_TITLE_URL` should be opt-in. 
I will supply it for the deployment of famf.app, but is unnecessary for everyone else.

Reply:
Currently if it is not set, the "Join at" div won't render, would you like me to remove famf.app and leave it as an empty string?

Reply:
Yeah empty string is good. I would only want it to display when people (like myself) deploy it to an actual URL.

Probably a good idea to update the readme on these settings.